src/App.vue

```diff
@@ -6,6 +6,7 @@
         :visible="menuVisibility"
         :on-select="goTo"
         :is-about-page-opened="isAboutPage"
+        @openDownloadDialog="openDownloadDialog"
```

Comment:
The good convension is to use kebab case for custom events, like

```
$emit('open-download-dialog)

@open-download-dialog=""
```

https://eslint.vuejs.org/rules/custom-event-name-casing.html

but looks like we haven't this rule in our eslint